services/src/test-library/src/main/java/org/openkilda/testing/service/lockkeeper/LockKeeperService.java

```diff
@@ -15,7 +15,7 @@
 
 package org.openkilda.testing.service.lockkeeper;
 
-import org.openkilda.messaging.model.SwitchId;
+import org.openkilda.testing.model.topology.TopologyDefinition.Switch;
```

Comment:
Don't quite understand this change. Why pass the whole Switch object?

Reply:
We need only switch name to manipulate its state. But I think it's more clear when we pass whole switch object